docs/index.rst

```diff
@@ -0,0 +1,28 @@
+TorchServe
+==================================
+
+TorchServe is a flexible and easy to use tool for serving PyTorch models.
+
+.. warning ::
+     TorchServe is experimental and subject to change.
+
+
+Basic Features
+----------------
+
+* `Serving Quick Start <https://pytorch.org/serve/install.html#serve-a-model>`_ - Basic server usage tutorial
```

Comment:
I'm thinking all of these can be relative links.

Reply:
Updated to relative link